docs/build-linux.md

```diff
@@ -1,6 +1,185 @@
+dependencies.
+
+Using system libraries shortens the build time, does not require Internet
+connection, and is easier if your distro supplies all the dependencies.
+
+## Packaging
+
+To create a distro package, use the [system libraries](#building-using-system-libraries)
+build method. 
+
+DOSBox Staging ships some binary files as a part of it's assets:
+
+- `contrib/resources/drives/y` - important DOS commands which are not yet
+  implemented internally.
+  These are pre-build DOS executables, taken from the FreeDOS or DOSBox-X
+  projects. To rebuild them, one might need legacy build tools, which are
+  considered exotic today (like the OpenWatcom compiler), that's why they are
+  not being compiled at build time.
+  If shipping such binaries is against your distro policy, feel free to strip
+  them away. DOSBox Staging will continue to work normally - the only real
+  disadvantage for the end user is that he will have to provide them by himself
+  to run some game/software installers.
+
+- `contrib/resources/freedos-cpi` - DOS screen fonts (CodePage Information
+  files, CPI).
+  These are a bitmap fonts in a native MS-DOS format, taken from the FreeDOS
+  project, painted by hand using a specialized font editor (so nothing like a
+  'source code' exists for them).
+  Probably all the other DOSBox forks uses such files in some form; it's just
+  most of them store the files content as an array of binary data, like the
+  [original DOSBox does](https://sourceforge.net/p/dosbox/code-0/HEAD/tree/dosbox/tags/RELEASE_0_74_3/src/dos/dos_codepages.h).
+  Running DOSBox Staging without these files is completely unsupported; even if
+  it seems to work for you, the internationalization features will malfunction.
+
+- `contrib/resources/freedos-keyboard` - DOS keyboard layout definitions.
+  Despite their extension suggestin a DOS device driver, these are data files,
+  not executables. They are, too, taken from the FreeDOS project; the binaries
+  were created from the source `*.KEY` text files, using a specialized tools,
+  written in Pascal - they are, too, part of the FreeDOS project, search for
+  `KEYB200S.ZIP`, `KEYB200X.ZIP`, `KC200S.ZIP`, and `KC200X.ZIP` files.
+  Probably all the other DOSBox forks uses such files in some form; it's just
+  most of them store the files content as an array of binary data, like the
+  [original DOSBox does](https://sourceforge.net/p/dosbox/code-0/HEAD/tree/dosbox/tags/RELEASE_0_74_3/src/dos/dos_keyboard_layout_data.h).
+  Running DOSBox Staging without these files is completely unsupported; even if
+  it seems to work for you, the internationalization features will malfunction.
+
+## Building using system libraries
+
+These are generic, distro-independent building instructions.
 
-## Minimum build requirements
+1. Install the necessary build tools:
+
+- GCC or Clang compiler (the compiler has to support C++20)
+- GIT
+- CMake
+- pkg-config
+
+2. Install the dependencies, development packages are needed too:
```

Comment:
Maybe put that in parens:

"(development packages are needed, too)"

Reply:
Changed.